### ISSUE TITLE
Feature Request: Tor Hidden Services

### ISSUE THREAD

**Issue by PowerPress** [stealth/drops#4]
Love the idea of this but could you incorporate using Tor Hidden Services to send and receive messages from the drops? This would prevent even more metadata leakage and really enhance the security of the platform. Additionally allow the creation of Tor Hidden Services so 2 users could directly send a p2p message to each other if desired via using Tor Hidden Services instead of using their own personal drops if desired.

**Comment by stealth** [comment]
absolutely. At least as an option. It shouldnt depend on tor. Yet, I know nothing about hidden services,
so maybe this would be a good starter project for me. But thats a topic of research on
how good this would match with the time stamps and vanishing of messages etc.

**Comment by PowerPress** [comment]
Now with version 3 of Tor hidden services being available could this be an option?
